loc.sh

```
@@ -0,0 +1,139 @@
+total_test_lines=0
+total_nontest_lines=0
+total_all_lines=0
+
+# Associative arrays for storing results
+declare -A test_lines_by_type
+declare -A nontest_lines_by_type
+declare -A total_lines_by_type
+
+# Calculate lines for each file type
+for ext in "${FILE_TYPES[@]}"; do
+    read test_count nontest_count <<< $(count_test_vs_nontest "$ext")
```

Shellcheck (0.10.0)

[warning] Quote this to prevent word splitting.

(SC2046)